src/libs/tools/lighttable.c

```diff
@@ -110,6 +116,22 @@ static void _lib_lighttable_update_btn(dt_lib_module_t *self)
     gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(w), (w == active));
     gtk_widget_queue_draw(w); // force redraw even if state not changed
   }
+  // should the culling restricted button also be active?
+  gtk_widget_set_visible(
+	d->layout_culling_restricted,
+	(d->layout == DT_LIGHTTABLE_LAYOUT_CULLING ||
+	 d->layout == DT_LIGHTTABLE_LAYOUT_CULLING_RESTRICTED)
+  );
+
+  if(d->layout == DT_LIGHTTABLE_LAYOUT_CULLING_RESTRICTED)
+    active = d->layout_culling_restricted;
+
+  children = gtk_container_get_children(GTK_CONTAINER(d->zoom_box));
+  for(GList *l = children; l; l = g_list_delete_link(l, l))
+  {
+    GtkWidget *w = (GtkWidget *)l->data;
+    gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(w), (w == active));
```

Comment:
This give an error : `Gtk-CRITICAL **: 22:15:06.321: gtk_toggle_button_set_active: assertion 'GTK_IS_TOGGLE_BUTTON (toggle_button)' failed` each time you press `xè shortcut for example.
This due to the fact that `d->zoom_box` contains other widgets than just toggle buttons...